app/src/main/java/com/eveningoutpost/dexdrip/EditAlertActivity.java

```diff
@@ -589,7 +589,7 @@ public void onClick(View v) {
         buttonalertMp3.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
                 AlertDialog.Builder builder = new AlertDialog.Builder(mContext);
-                builder.setTitle("What type of Alert?")
+                builder.setTitle("")
```

Comment:
What is the idea of removing the title?
I suggest keeping the title but creating a translatable resource string of it. Default English title could be "Select type of sound" or "Select sound source" or something similar.

Reply:
Thanks for the review.
I have followed your advice.
In the process discovered what "Reminders" were.  Made the same change to reminders.